### PR TITLE
Comments on selectors; Added GitHub and GitLab style

### PR DESCRIPTION
**Pull request by mattalexx**

### CODE REVIEW

File: src/generators/text-style.ts [hunk 1]
```diff
@@ -1,10 +1,27 @@
 import type {FilterConfig} from '../definitions';
 
 export function createTextStyle(config: FilterConfig): string {
-    const lines: string[] = [];
     // Don't target pre elements as they are preformatted element's e.g. code blocks
     // Exclude font libraries to preserve icons
-    lines.push('*:not(pre, pre *, code, .far, .fa, .glyphicon, [class*="vjs-"], .fab, .fa-github, .fas, .material-icons, .icofont, .typcn, mu, [class*="mu-"], .glyphicon, .icon) {');
+
+    const mono: string = [
```
Comment (Gusted) [inline src/generators/text-style.ts:7]:
```suggestion
    const exlcudingSelectors: string = [
```
Should be a bit more verbose. `mono` is vague to me.

File: src/generators/text-style.ts [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+
+        // GitHub
+        '.blob-code-inner, .blob-code-inner *',
+
+        // Gitlab
+        '.monaco-editor',
+    ].join(', ');
+
+    const lines: string[] = [];
+    lines.push(`*:not(${ mono }) {`);
```
Comment (Gusted) [inline src/generators/text-style.ts:24]:
```suggestion
    lines.push(`*:not(${ exlcudingSelectors }) {`);
```

**Review by Gusted — APPROVED** [review]
Thanks!

**Comment by mattalexx** [comment]
Thank you for saving my eyes.

**Pull request merged**